frontend/packages/react-app/.env.example

```diff
@@ -1,5 +1,6 @@
-REACT_APP_RPC_URL_1=
-REACT_APP_RPC_URL_4=
+REACT_APP_RPC_URL=
 REACT_APP_FORTMATIC_API_KEY=
 REACT_APP_PORTIS_DAPP_ID=
 REACT_APP_TARGETS_UPPER_LIMIT=2000
+REACT_APP_SUBGRAPH=
+REACT_APP_CHAIN_ID=
```

Comment:
could you add following envs to env.example? 

```
REACT_APP_CONTRACT_MIGRATIONS,
REACT_APP_CONTRACT_WALLETDISTRIBUTOR,
REACT_APP_CONTRACT_UUIDDISTRIBUTOR,
REACT_APP_CONTRACT_CCTWALLETDISTRIBUTOR,
```

Reply:
(without values)

Reply:
Sorry for my mistake!